common/meta/api/src/schema_api.rs

```diff
@@ -41,6 +41,8 @@ use common_meta_types::UndropDatabaseReply;
 use common_meta_types::UndropDatabaseReq;
 use common_meta_types::UndropTableReply;
 use common_meta_types::UndropTableReq;
+use common_meta_types::UpdateTableMetaReply;
+use common_meta_types::UpdateTableMetaReq;
```

Comment:
Is this gonna replace the UpsertTableOption RPC?

Reply:
> Is this gonna replace the UpsertTableOption RPC?

I have no idea, this type is not from this PR, may be ask @dantengsky

Reply:
Yes, `update_table_meta` is added in https://github.com/datafuselabs/databend/pull/5476.